tools/sdk/lib/fix_sdk_libs.sh

```diff
@@ -1,36 +1,67 @@
 #!/bin/bash
 set -e
 
-export PATH=../../xtensa-lx106-elf/bin:$PATH
+export PATH=../../../xtensa-lx106-elf/bin:$PATH
 VERSION=$(basename ${PWD})
 
 addSymbol_system_func1() {
-	ADDRESS=$1
-	xtensa-lx106-elf-objcopy --add-symbol system_func1=.irom0.text:${ADDRESS},function,global user_interface.o
+	if ! xtensa-lx106-elf-nm  user_interface.o | grep -q " T system_func1"; then # Don't add symbol if it already exists
```

Comment:
Sorry, I just noticed that `grep -q` will always return 0.

Reply:
Even if it's not in there yet? That's strange (be aware that it got already added twice or so...)

Reply:
Yes, that's what I noticed when I tried some modifications.  grep -q appears to be an exception on returning non-zero exit status.

Reply:
For me it works:
```
$ echo ThisIsATest | grep -q Test
$ echo ?$
0
$ echo ThisIsATest | grep -q Fail
$ echo $?
1
```

Reply:
Well, I am confused. It wasn't working for me and now it is. I had duplicate entries and the man page says `grep -q` exits with 0. However, I just tried everything again and it worked. 

Sorry, I don't know what happened. Just ignore me, I am going to take a break.

Reply:
The current version as it is in the repo at the moment is having duplicate entries, those should be removed at some point. So maybe that caused the confusion (and that's why I added that check)

Reply:
Hmm, strange. I only picked up the script and tried it and had duplicate entries. As well as other problems from running the script at the wrong directory which creates a `libmain.a` which creates a lot of grief.

Reply:
@Flole998 what do you mean "duplicate entries in the current version" ?
Is it in master or in this pull request ?

Reply:
In master (and also in this pull request because I didn't remove/fix it as that would be something for another PR I figured)